src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -289,14 +289,24 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
         if (issuanceMintAmount < _minAmountOut) {
             revert Module__BondingCurveBase__InsufficientOutputAmount();
         }
-        // Mint tokens to address
-        _mint(_receiver, issuanceMintAmount);
+
+        // Use abstract function to distribute issuance tokens
+        _distributeIssuanceToken(_receiver, issuanceMintAmount);
+
         // Emit event
         emit TokensBought(
             _receiver, _depositAmount, issuanceMintAmount, _msgSender()
         );
     }
 
+    /// @dev    Abstract function to distribute issuance tokens.
+    /// @param  _receiver The address that will receive the bought tokens.
```

Comment:
> [...] receive the **bought** tokens.

Is this really just for buying tokens? Didn't check, if so then that's okay probably, but I believe we'll likely also use this to distribute fees or whatnot.

Reply:
Fee transferal is already included in a different function
Had this in the sell section as well and changed it there
so i  must have forgot it here

Reply:
After discussing we agreed to name them
handleIssuanceTokenAfterBuy
handleCollateralTokenAfterSell